backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ir/Ir.kt

```diff
@@ -149,6 +149,16 @@ internal class KonanSymbols(context: Context, val symbolTable: SymbolTable, val
 
     val allIntegerClasses = signedIntegerClasses + unsignedIntegerClasses
 
+    val unsignedToSignedOfSameBitWidth = unsignedIntegerClasses.associate {
+        it to when (it) {
+            uByte -> byte
+            uShort -> short
```

Comment:
Hope we don't need chat -> short conversion, do we?

Reply:
We don't.